homework-submission/week1/src/stores/todosStore.js

```diff
@@ -0,0 +1,54 @@
+import { observable, action, configure } from 'mobx';
+import todoObj from '../todo.json';
+
+configure({
+  enforceActions: true
```

Comment:
The boolean value of this configuration is deprecated [mobx@5.1.0](https://github.com/mobxjs/mobx/blob/master/CHANGELOG.md#510), instead use 

    configure({
      enforceActions: 'observed'
    });